### DISCUSSION TITLE
Schedule automated backups to a .DASH file

### DISCUSSION THREAD

**Discussion by watchoutforjaws** [Dashlane/dashlane-cli#159] [Ideas]
Hello. I'd like to schedule an automated backup of my Dashlane password vault, to a .DASH file to store as part of an offsite backup, incase the service disappears etc. Advised may be possible through dashlane-cli. I'm using Mac OS / Firefox? Thanks.

**Comment by Mikescops** [comment]
Having a .DASH export feature in the CLI could be great indeed. That said it requires quite some work to make sure it has the proper format.

I think an intermediate solution would be to be able to export the local encrypted database of the CLI and be able to read it later.

**Reply by watchoutforjaws** [reply]
That sounds like a good idea - yes - the basic idea would be to have a backed up copy of passwords which could be accessed in case dashlane were to disappear as a company.

**Reply by Mikescops** [reply]
In the next release I included a command that exports the local vault to a chosen path. It's not following the .DASH format but it's a raw extract of the database with all your passwords (and other secrets) encrypted with your master password, which means that if Dashlane ever disappear you can still decrypt them 👍

**Comment by watchoutforjaws** [comment]
That is incredible - thank you so much Corentin 🙂

**Reply by Mikescops** [reply]
You're welcome, I added a short explanation on the documentation website: https://dashlane.github.io/dashlane-cli/personal/backup
